cmake/targets/common.cmake

```diff
@@ -24,7 +24,7 @@ endif()
 
 target_link_libraries(sunshine ${SUNSHINE_EXTERNAL_LIBRARIES} ${EXTRA_LIBS})
 target_compile_definitions(sunshine PUBLIC ${SUNSHINE_DEFINITIONS})
-set_target_properties(sunshine PROPERTIES CXX_STANDARD 17
+set_target_properties(sunshine PROPERTIES CXX_STANDARD 20
```

Comment:
It looks like not all targets are ready for C++20. Can you scope the C++20 change to either just the specific C++ file that require it, or just to the Windows platform?